disasterinfosite/models.py

```diff
@@ -333,80 +333,89 @@ def getRelatedTemplate(self):
     def findSnuggetsForPoint(lat=0, lng=0, merge_deform = True):
         pnt = Point(lng, lat)
 
-######################################################
-# GENERATED CODE GOES HERE
-# DO NOT MANUALLY EDIT CODE IN THIS SECTION - IT WILL BE OVERWRITTEN
-# modelsGeoFilters
```

Comment:
Oh, also: I'm not sure that import.py in its current form will fail gracefully if we try to use it to replace all the other generated code blocks, but I'm not too worried - if it doesn't, it won't be too much work to add the error checking so it does.